lightspeed/modules/proc_configure-vscode-extension-onpremise-deployment.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: PROCEDURE
+
+[id="configure-vscode-extension-onpremise-deployment_{context}"]
+
+= Configuring Ansible VS Code extension for {LightspeedShortName} on-premise deployment
+
+To access the on-premise deployment of {LightspeedShortName}, all Ansible users within your organization must install the Ansible Visual Studio (VS) Code extension in their VS Code editor, and configure the extension to connect to the on-premise deployment. 
+
+.Prerequisites
+
+* You have installed VS Code version 1.70.1 or later. 
+
+.Procedure
+
+. Open the VS Code application.
+. From the *Activity* bar, click the *Extensions* icon.
+. From the *Installed Extensions* list, select *Ansible*.
+. From the *Ansible* extension page, click the *Settings* icon (image:settings-icon-ansible-vscode-extension.png[Settings icon]) and select *Extension Settings*.
+. Select *Ansible Lightspeed* settings and specify the following information:
+* In the *URL for Ansible Lightspeed* field, enter the *Route URL* of the {LightspeedShortName} on-premise deployment. The Ansible users must have AAP controller credentials. 
```

Comment:
I'd remove "the" from before "Ansible users" so it reads "Ansible users must have AAP controller credentials." Also in that sentence, AAP should be an attribute, no?

Reply:
Done